src/wizards/topics.ts

```diff
@@ -65,7 +65,10 @@ export async function addTopicWizard(clientAccessor: ClientAccessor, clusterSett
             showErrorMessage(`Error while creating topic for cluster '${clusterName}'`, result[0].error);
         } else {
             explorer.refresh();
-            window.showInformationMessage(`Topic '${topic}' in cluster '${clusterName}' created successfully`);
+            const result = await window.showInformationMessage(`Topic '${topic}' in cluster '${clusterName}' created successfully`, 'Select', 'Close');

```

Comment:
I would simply select the created topic. I think adding buttons bloats the UI

Reply:
fixed with a setTimeout